spec/es_spec_helper.rb

```diff
@@ -9,57 +9,61 @@
 CONTAINER_IMAGE = "elasticsearch"
 CONTAINER_TAG = "1.6"
 
-module ESHelper
+DOCKER_INTEGRATION = ENV["DOCKER_INTEGRATION"]
```

Comment:
should we also allow environment variables to explicitly target an ES instance with custom host/port?

this can wait for another follow-up PR if we think its a useful option